app/views/includes/sending-evidence.html

```diff
@@ -0,0 +1,19 @@
+{% set reference = "SC242/16/03435" %}
```

Comment:
If this is coming from the api, maybe it could be added to the mock data?

Reply:
At this stage we don't know which API call it would come from so I'm not sure it's worth guessing.